resources/public/api/conf/4.0/schemas/broughtForwardLosses/listBFLoss/list_response.json

```diff
@@ -54,40 +54,6 @@
 			"pattern": "^2[0-9]{3}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]{0,3}|)Z$",
 			"example": "2021-07-13T12:13:48.763Z"
 		  },
```

Comment:
There's some trailing commas, in each of these responses. Can tidy up? It might be invalid json as-is; did the OAS still render ok locally?